BlackMarket/Views/Dashboard/PaymentMethods.swift

```diff
@@ -0,0 +1,57 @@
+//
+//  PaymentMethods.swift
+//  BlackMarket
+//
+//  Created by Santiago Andracchi on 22/09/2023.
+//
+
+import SwiftUI
+
+struct PaymentMethodsView: View {
+  var body: some View {
+    VStack(alignment: .center) {
+      Text(LocalizedString.PaymentMethods.title)
+        .bold()
+      HStack(alignment: .top) {
+        VStack {
+          Image("blackmarket_credit_card")
+          Text(LocalizedString.PaymentMethods.credit)
+            .padding(.top, 14)
+        }
+        Image("blackmarket_line")
```

Comment:
we should use the image name constants right?
Another option is to add an Image extension:
```
extension Image {
  static let creditCard = Image("blackmarket_credit_card")
}
```

and you use it like:`Image.creditCard`

Reply:
Yes! I did it on `PaymentMethodsView`. I don't know why files are duplicated. Merge conflicts, I guess. I will remove this file, but I will also apply your suggestion. Thanks!